AsyncDisplayKit/ASMultiplexImageNode.mm

```diff
@@ -458,32 +485,34 @@ - (UIImage *)_bestImmediatelyAvailableImageFromDataSource:(id *)imageIdentifierO
  */
 - (void)_updateProgressImageBlockOnDownloaderIfNeeded
 {
-    // Read our interface state before locking so that we don't lock super while holding our lock.
```

Comment:
Most of these changes are indentation sadly :/

New / edited lines are 488 & 499 (on the right).